tst/standard/grape.tst

```diff
@@ -41,10 +48,11 @@ true
 gap> GeneratorsOfCayleyDigraph(digraph);
 [ () ]
 gap> digraph := CayleyDigraph(group, [(1, 2, 3, 4), (2, 5)]);
-Error, the 2nd argument <gens> must consist of elements of the 1st argument,
+Error, the 3rd argument (a homog. list) must consist of elements of the 2nd ar\
+gument (a list)
 gap> group := FreeGroup(2);;
 gap> digraph := CayleyDigraph(group);
-Error, the 1st argument <G> must be a finite group,
+Error, the 2nd argument (a group) must be finite
```

Comment:
It's fine though, since in practice you get
```
gap> digraph := CayleyDigraph(group);
Error, the 2nd argument (a group) must be finite at /Users/Wilf/gap/pkg/digraphs/gap/grape.gi:123 called from
CayleyDigraphCons( IsImmutableDigraph, G, GeneratorsOfGroup( G )
 ) at /Users/Wilf/gap/pkg/digraphs/gap/grape.gi:155 called from
```